packages/neuron-wallet/scripts/notarize.js

```diff
@@ -6,6 +6,11 @@ exports.default = async function notarizing(context) {
     return
   }
 
+  if (process.env.SKIP_NOTARIZE === 'true') {
+    console.warn('Skip notarizing when package on pull_request')
```

Comment:
The condition doesn't cover if the package is triggered by a pull_request, so the message is misleading.